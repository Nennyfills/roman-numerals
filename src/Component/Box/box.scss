.box{
  border-radius: 10px;
  padding: 1rem;
  margin: 2rem;
  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
  width: 30%;
}
