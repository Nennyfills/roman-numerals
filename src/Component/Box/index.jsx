import React from 'react';
import PropTypes from 'prop-types';

import './box.scss';

const Box = ({ children, boxStyles }) => (
  <div className={`box ${boxStyles || ''}`}>{children}</div>
);

Box.defaultProps = {
  children: <div />,
  boxStyles: '',
};
Box.propTypes = {
  children: PropTypes.element,
  boxStyles: PropTypes.string,
};
export default Box;
