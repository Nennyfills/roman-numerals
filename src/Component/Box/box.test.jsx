import React from 'react';
import { render } from '@testing-library/react';

import Box from './index';

test('renders a properly', () => {
  const { container } = render(<Box />);
  expect(container.firstChild).toMatchSnapshot(`
    <div />
  `);
});
