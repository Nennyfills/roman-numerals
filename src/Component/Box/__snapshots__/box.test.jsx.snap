// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a properly: 
    <div />
   1`] = `
<div
  class="box "
>
  <div />
</div>
`;
